webpack.config.dev.js

```diff
@@ -35,12 +35,22 @@ module.exports = {
         loader: ExtractTextPlugin.extract({
           fallback: 'style-loader',
           use: [
-            'css-loader?modules&importLoaders=1&localIdentName=[path][local]__[hash:base64:5]',
+            'css-loader?modules&importLoaders=2&localIdentName=[path][local]__[hash:base64:5]',
```

Comment:
what does this do?

Reply:
According to the docs, this is the number of loaders _following_ `css-loader`.